app/views/view_data/departments/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <main id="content" role="main">
 
-  <%= breadcrumbs %>
+  <a class="link-back" href="<%= view_data_department_metrics_path(@department) %>">Back to <%= @department.name %> list view</a>
```

Comment:
I thought it might be good to change 'list view' to 'service data'?

Reply:
https://sp-trends.herokuapp.com/a-separate/org-environment-agency-detail says list view, but I guess it needs a content review as well at some point.

Reply:
👍